server.go

```diff
@@ -190,7 +218,21 @@ func (s *server) Serve() chan error {
 	// Strip prefix URI since DelegatedTranslator uses a nested mux.
 	mux.Handle("/routing/v1/", http.StripPrefix("/routing/v1", delegated))
 
-	mux.Handle("/", s)
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		// Do not fall back on web-ui on unknown paths. Instead, strictly check the path and
+		// return 404 on anything but "/" and "index.html". Otherwise, paths that are supported by
+		// some backends and not others, like "/metadata" will return text/html.
+		switch r.URL.Path {
+		case "/", "/index.html":
+			if r.Method == http.MethodGet {
```

Comment:
should we also allow head / options?

Reply:
https://github.com/ipni/indexstar/issues/114